sdk/communication/azure-communication-sms/azure/communication/sms/_generated/_configuration.py

```diff
@@ -38,7 +38,7 @@ def __init__(
         super(AzureCommunicationSMSServiceConfiguration, self).__init__(**kwargs)
 
         self.endpoint = endpoint
-        self.api_version = "2020-07-20-preview1"
+        self.api_version = "2021-03-07"
```

Comment:
We should double check if we are going to remove preview now or later

Reply:
We discussed this offline and decided to leave it alone.